### PR TITLE
Worked on two functions

### PR DESCRIPTION
**Pull request by alekshere**
As the title says, I identified untested regions of the code, selected two functions with multiple conditional paths, instrumented these functions to measure coverage, and created test cases to cover the selected functions achieving 100% branch coverage.

Files with functions I worked on:

telegram/ext/_handlers/conversationhandler.py
telegram/ext/_handlers/conversationhandler.py
Files with test cases I worked on:

tests/test_message.py
tests/ext/test_conversationhandler.py

### CODE REVIEW

**Comment by EzraBerendsen** [comment]
Please add .coverage to the .gitignore, so we don't all have to pull in those files. Thanks in advance.

**Review by EzraBerendsen — CHANGES_REQUESTED** [review]
Please add .coverage to the .gitignore, so we don't all have to pull in those files. Thanks in advance.

File: telegram/ext/_handlers/conversationhandler.py [hunk 1]
```diff
@@ -111,12 +120,75 @@ def resolve(self) -> object:
...
+            branch_coverage["104"] = True
             # returning None from a callback means that we want to stay in the old state
             return self.old_state
 
         return res
+    
+    """ def print_coverage():
+        for branch, hit in branch_coverage.items():
+            print(f"{branch} was {'hit' if hit else 'not hit'}") """
+
```
Comment (EzraBerendsen) [inline telegram/ext/_handlers/conversationhandler.py:135]: Probably want to remove this?

File: telegram/ext/_handlers/conversationhandler.py [hunk 2]
```diff
@@ -111,12 +120,75 @@ def resolve(self) -> object:
...
+
+
+
+
+class ConversationHandler:
+    END = "end"
```
Comment (EzraBerendsen) [inline telegram/ext/_handlers/conversationhandler.py:141]: Don't think this is needed

File: telegram/ext/_handlers/conversationhandler.py (hunk 1 above)
Comment (alekshere) [reply telegram/ext/_handlers/conversationhandler.py:135]: It is needed for the assignment presentation.

File: telegram/ext/_handlers/conversationhandler.py (hunk 2 above)
Comment (alekshere) [reply telegram/ext/_handlers/conversationhandler.py:141]: It is needed for the testcase.

File: telegram/ext/_handlers/conversationhandler.py (hunk 1 above)
Comment (EzraBerendsen) [reply telegram/ext/_handlers/conversationhandler.py:135]: why is it commented?

**Comment by EzraBerendsen** [comment]
LGTM.

**Pull request merged**